README.md

```diff
@@ -20,9 +20,7 @@ bundle install
 touch .env
 
 # Clone production data
-rake db:create db:migrate
-rake cascade:sync
-rake faculty:sync
+cap production db:clone
```

Comment:
Why is this a cap command? I associate `cap` with deployment. This doesn't seem related to deployment to me but with setting up developer's env.